tools/engine_tool/lib/src/commands/flags.dart

```diff
@@ -16,6 +16,10 @@ const String builderFlag = 'builder';
 const String concurrencyFlag = 'concurrency';
 const String configFlag = 'config';
 const String dryRunFlag = 'dry-run';
+const String execStrategyFlag = 'exec-strategy';
```

Comment:
One last suggested change, is to `s/exec/build`. The reason is we don't have the concept of an "execution strategy", that is, something like `et test //foo/bar` will _always_ run the test locally today.

Reply:
Done.